### DISCUSSION TITLE
Unable to submit ticket, error with github sign on on main page and login connections

### DISCUSSION THREAD

**Discussion by NathanScott85** [vercel/community#4703] [Account Issue] [answered]
### Question

The first issue is when I try submitting a support ticket I get the following error:

![image](https://github.com/vercel/community/assets/5598350/af59c291-f648-49e0-b8b8-0f2ba169487e)

The second issue when you try to click on the GitHub login button on Vercel login page nothing happens, there are errors in the console.

Third issue I am unable to connect my account through login connections to GitHub which is stuck on infinite loading.

![image](https://github.com/vercel/community/assets/5598350/d7a975c6-629a-4982-88ca-ea3cb665783c)

Also locked out of having feature branches, which I assume is because of the above.

**Comment by sebastianroming** [comment, marked as answer]
Hi @NathanScott85,

It looks like you were able to create the support case and our team already responded. Let's continue investigating your issue there.

Thank you!
